torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8825,8 +8825,10 @@ def __init__(
         arity: int = 3,
         rightmost_supports_scalar: bool = False,
         rightmost_supports_scalarlist: bool = False,
+        without_values_kwarg: bool = False,
     ):
         super().__init__(3 + 1, True, True)
```

Comment:
Preexisting issue, but this looks wrong? It's not forwarding the args

Reply:
what about the arity?

Reply:
Should the default value be `4`?

Reply:
the arity is not being passed to super...

Reply:
Right. It's sending `3 + 1` to super and I'm not sure what the correct value should be.

Reply:
This function has an arity parameter, just forward it?

Reply:
Fixed now